benchmark/benchmark_experimental_vocab.py

```diff
@@ -6,8 +6,8 @@
 from torchtext.experimental.datasets import AG_NEWS
 from torchtext.experimental.vocab import (
     vocab as VocabExperimental,
-    vocab_from_file,
-    vocab_from_raw_text_file
+    load_vocab_from_file,
+    build_vocab_from_raw_text_file
```

Comment:
Let's change "build_vocab_from_raw_text_file" to "build_vocab_from_text_file" until we have a better definition of raw